### DISCUSSION TITLE
Hugo Blowfish date and time

### DISCUSSION THREAD

**Discussion by darreecool** [nunocoracao/blowfish#1489] [Q&A]
Hello so im facing a problem. blowfish is always making a structure that groups stuff by year. they have a command called “groupByYear” but i want to find a way so that it groups by year only if i specify a year. if not i dont want anything to show up. Here is an example in the first photo since i did not specify a year it is showing 0001 but id like to remove that without removing the year in the second photo
![image_2024-05-23_151000505](https://github.com/nunocoracao/blowfish/assets/32749292/6f89c3f5-9f26-4edb-9222-34bb59d1153e)
![image_2024-05-23_151008084](https://github.com/nunocoracao/blowfish/assets/32749292/5cfb49d9-9a23-42c4-8bb6-a811409945be)

**Comment by darreecool** [comment]
someone please help

**Reply by mnjm** [reply]
I think you can use [List Shortcode](https://blowfish.page/docs/shortcodes/#list) a possible work around. @darreecool

**Reply by darreecool** [reply]
i found a good shortcode "Timeline" but i dont understand how to get it to work? i get this error when i try pasting "shortcode: shortcode "timeline" must be closed or self-closed " this is what i pastedD
![image_2024-06-03_124604272](https://github.com/nunocoracao/blowfish/assets/32749292/e1ece654-cc75-4b7b-83a9-9634f30beebf)

**Reply by mnjm** [reply]
You should close the `{{< timeline >}}` that you've opened in the first line.

**Reply by darreecool** [reply]
yo thanks for the help it was really helpful i got it working as i wanted it. but i was wondering is there a way to limit the amount of text that is shown in a "timeline" and maybe add a "Read More" button to it to expand for the rest of the text?

**Reply by darreecool** [reply]
would that be possible or nah?

**Reply by mnjm** [reply]
Artificially create a "next" page and add a "Next" [button](https://blowfish.page/docs/shortcodes/#button)¯_(ツ)_/¯. This might work, but would not be clean, or better inject custom html. 

I think Blowfish offers only blog layouts, although I have not gone through half of what Blowfish can do.

**Reply by darreecool** [reply]
i just saw something using timeline as a work around got some problems. the * and ### dont work nether does <br> inside of timelines
